pkg/exporter/ovs.go

```diff
@@ -152,9 +180,14 @@ func (m *OvsMonitor) handleOvsUpdates(updates ovsdb.TableUpdates) {
 					m.collectorCache.AddIface(generateInterface(row.New))
 				}
 				if table == "Bridge" {
-					if FilterBridge(row.New.Fields["name"]) {
+					if FilterBridge(row.New.Fields["name"], datapath.POLICY_BRIDGE_KEYWORD) ||
+						FilterBridge(row.New.Fields["name"], datapath.CLS_BRIDGE_KEYWORD) {
 						continue
 					}
+					if !FilterBridge(row.New.Fields["name"], INTERNAL_BRIDGE_KEYWORD) &&
+						FilterNormalBridge(row.New.Fields["name"]) && reflect.DeepEqual(row.Old, empty) {
```

Comment:
If we transfer bridge mode from chain to single, row.Old will not be empty.

Reply:
this filter condition has been removed